webpack.config.coffee

```diff
@@ -20,7 +20,7 @@ else
 module.exports =
   cache: true
 
-  devtool: 'source-map' # Always include source maps!
+  devtool: '#inline-source-map' # Always include source maps!
 
```

Comment:
I don't think we can use `inline-source-map`.  https://webpack.github.io/docs/configuration.html says they'll be included as a `DataUrl`, which I believe means they're base64 encoded and included in the actual file.

If we do that then I **think** production will include the source maps in the build for all users, making the download size larger.  Currently it's supposed to be a URL to the source map, which is only download when the dev tools source console is opened.

Maybe I'm incorrect though.  We should probably deploy to dev to test